### DISCUSSION TITLE
Log4j vulnerability

### DISCUSSION THREAD

**Discussion by ken8521** [ONLYOFFICE/DocumentServer#1555] [Q&A] [answered]
Just want to verify if this container is affected by this vulnerability. Best I can tell it is not, but just want to be sure.

**Comment by ShockwaveNN** [comment, marked as answer]
Hi, we do not use `log4j` in any form in DocumentServer, so we are safe

More details in [our blog](https://www.onlyoffice.com/blog/2021/12/apache-log4j-security-bulletin-stay-safe-with-onlyoffice/)
